lm_eval/tasks/score/NON_GREEDY.md

```diff
@@ -0,0 +1,32 @@
+```
+Copyright (c) 2024, NVIDIA CORPORATION & AFFILIATES.  All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+````
```

Comment:
add more details on the non-greedy eval? 5 seeds various tasks, fixed prompt, supported datasets? expected result? i.e. we don't want variance based on the seed alone
temp or other inference params that are worth mentioning?

Reply:
you have a high level overview of the task but I'd repeat it here as some details are still missing or expand the one in the README and put a reference here.